backend/src/api/channels/channels.service.ts

```diff
@@ -127,7 +127,7 @@ export class ChannelsService {
 
     const newMessage: Message = {
       id: this.messageMap.size,
-      senderId: 0,
+      senderId: undefined,
```

Comment:
```suggestion
      senderId: null,
```
명시적인 빈 값은 null로 잡아주세욥